### PR TITLE
[DNM] meta collection integration jobs

### PR DESCRIPTION
**Pull request by rohitthakur2590**
Signed-off-by: Rohit Thakur <rohitthakur2590@outlook.com>
Depends-On:  https://github.com/ansible/network-ee/pull/119

### CODE REVIEW

File: zuul.d/project-templates.yaml [hunk 1]
```diff
@@ -1741,3 +1741,20 @@
...
       jobs:
         - noop
+- project-template:
+    name: ansible-collections-ansible-network-resource_manager-cisco-ios
+    check:
+      jobs: &ansible-collections-ansible-network-cisco-ios-jobs
+        - ansible-test-network-integration-ansible-network-resource_manager-ios-python36:
+            vars:
+              ansible_test_collections: true
+            voting: false
```
Comment (GomathiselviS) [inline zuul.d/project-templates.yaml:1751]: Why is the job non-voting?

**Comment by pabelanger** [comment]
@rohitthakur2590 can you build these a top of network-ee-integration-tests-vyos? These will be our new execution environment integration tests moving forward.

**Comment by rohitthakur2590** [comment]
> @rohitthakur2590 can you build these a top of network-ee-integration-tests-vyos? These will be our new execution environment integration tests moving forward.

@pabelanger The tests we have in the network collection are for ios for now, can we keep it for ios instead vyos?

**Comment by rohitthakur2590** [comment]
network-ee-integration jobs added (ref: https://github.com/ansible/ansible-zuul-jobs/pull/1066)

**Comment by rohitthakur2590** [comment]
recheck

**Pull request closed without merging**